connectors/agentic-ai/AI_AGENT.md

```diff
@@ -146,6 +146,7 @@ leading to the following result
     } ],
     "metadata" : {
       "framework" : {
+        "finishReason" : "STOP",
```

Comment:
Generated from the response samples - I missed re-building after the last adaption.